connector/src/main/java/org/geysermc/connector/network/translators/bedrock/BedrockAnimateTranslator.java

```diff
@@ -61,13 +57,13 @@ public void translate(AnimatePacket packet, GeyserSession session) {
             // These two might need to be flipped, but my recommendation is getting moving working first
             case ROW_LEFT:
                 // Packet value is a float of how long one has been rowing, so we convert that into a boolean
-                isSteeringLeft = packet.getRowingTime() > 0.0;
-                ClientSteerBoatPacket steerLeftPacket = new ClientSteerBoatPacket(isSteeringRight, isSteeringLeft);
+                session.setSteeringLeft(packet.getRowingTime() > 0.0);
+                ClientSteerBoatPacket steerLeftPacket = new ClientSteerBoatPacket(session.isSteeringLeft(), session.isSteeringRight());
```

Comment:
Why are these values reversed? In the protocol, the first boolean is the right paddle turning, and the second is the left. Might be worth noting this if this is intentional.

Reply:
https://wiki.vg/Protocol#Steer_Boat MCProtocolLib appears to be incorrect here.